File: ios/Classes/ble/models/BleMatchDevice.swift
//
//  BleMatchDevice.swift
//  flutter_ezw_ble
//
//  Created by Whiskee on 2025/1/13.
//

struct BleMatchDevice: Codable {
    let sn: String
    let devices: [BleDevice]
    
    var belongConfig: String? {
        get {
            return devices.first?.belongConfig
        }
    }
}

File: android/src/main/kotlin/com/fzfstudio/ezw_ble/ble/models/BleMatchDevice.kt
package com.fzfstudio.ezw_ble.ble.models

import com.fzfstudio.ezw_utils.gson.GsonSerializable

data class BleMatchDevice(
    val sn: String,
    val devices: List<BleDevice>,
): GsonSerializable() {

    val belongConfig: String
        get() = devices.first().beLongConfig

}
